src/sentry/tasks/post_process.py

```diff
@@ -22,6 +23,16 @@
 logger = logging.getLogger('sentry')
 
 
+def _get_service_hooks(project_id):
+    from sentry.models import ServiceHook
+    cache_key = 'servicehooks:1:{}'.format(project_id)
+    result = cache.get(cache_key)
+    if result is None:
+        result = ServiceHook.objects.filter(project_id=project_id).values('id', 'events')
```

Comment:
Coerce this to a `list()` before caching, otherwise you're caching an unevaluated QuerySet.